hidden/Features/Preferences/PreferencesViewController.swift

```diff
@@ -114,6 +116,20 @@ class PreferencesViewController: NSViewController {
         }
     }
     
+    @IBAction func enablePermHideChange(_ sender: NSButton) {
+        switch sender.state {
+        case .on:
+            Util.setEnablePermHide(true)
+            Util.permHideChanged()
+        case .off:
+            Util.setEnablePermHide(false)
+            Util.permHideChanged()
+        default:
+            break
+        }
+    }
+    
+    
```

Comment:
for UI action, I prefer readable with action descriptions naming like "btnPermClicked". With this, we know this control is button for Perm feature and the action of button is clicked